feature/networkinstance/networkinstance.go

```diff
@@ -52,6 +53,12 @@ func (ni *NetworkInstance) validate() error {
 	return ni.oc.Validate()
 }
 
+// WithStaticRoute sets the prefix value for static route.
+func (ni *NetworkInstance) WithStaticRoute(prefix string) *NetworkInstance {
```

Comment:
I think this API should be

WithStaticRoute(prefix string, nextHops []string)

I think we want to add all the next hops for a given prefix in the same API.

Reply:
If a prefix with 0 next-hops is invalid then we should update the validate() method to check for this as fail in the AugmentDevice() method. See the BGP example for something similar.

Reply:
Done!